### PR TITLE
Add image for an inbox item

### PR DESCRIPTION
**Pull request by ftcvlad**
### Description of Changes

1) Set imagePath from inboxConfig when create KSInAppInboxItem
2) Add `(NSURL* _Nullable) getImageUrl:(int)width;` and `(NSURL* _Nullable) getImageUrl;` public methods to get a formatted url.

### Breaking Changes

-   None

### Release Checklist

Prepare:

-   [x] Detail any breaking changes. Breaking changes require a new major version number
-   [x] Check `pod lib lint` passes
-   [x] Check all targets (iOS, extension, macOS, statics) build
-   [x] Install branch via Cocoapods into empty project & verify can import SDK (only needed if adding/removing files)

Bump versions in:

-   [x] `Sources/Kumulos+Stats.m`
-   [x] `KumulosSdkObjectiveC.podspec`
-   [x] `KumulosSdkObjectiveCExtension.podspec`
-   [x] All relevant build targets
-   [x] `README.md`

Release:

-   [ ] Squash and merge to master
-   [ ] Delete branch once merged
-   [ ] Create tag from master matching chosen version
-   [ ] Run `pod trunk push` to publish to CocoaPods

### CODE REVIEW

File: Sources/KumulosInApp.m [hunk 1]
```diff
@@ -6,14 +6,24 @@
...
+@interface KSInAppInboxItem()
+
+@property (nonatomic,readonly) NSString* _Nonnull imagePath;
+@property (nonatomic,readonly) NSDate* _Nullable readAt;
+
+@end
+
 @implementation KSInAppInboxItem
 
+int const DEFAULT_IMAGE_WIDTH = 200;
```
Comment (cgwyllie) [inline Sources/KumulosInApp.m:18]: I know this is the figure we've been arbitrarily throwing around, for modern screens will this be a sensible default and produce clear images?

File: Sources/KumulosSDKExtension/KumulosNotificationService.m [hunk 2]
```diff
@@ -13,10 +13,10 @@
 #import "../Shared/KumulosSharedEvents.h"
 #import "../Shared/KSAppGroupsHelper.h"
 #import "../Shared/KSPendingNotificationHelper.h"
+#import "../Shared/KSMediaHelper.h"
```
Comment (cgwyllie) [inline Sources/KumulosSDKExtension/KumulosNotificationService.m:16]: Missing from pod for ext or included by glob?

File: Sources/Shared/KSMediaHelper.m [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+NSString* const _Nonnull KSMediaResizerBaseUrl = @"https://i.app.delivery";
+
++ (NSURL *) getCompletePictureUrl:(NSString *)pictureUrl width:(NSUInteger)width {
+    if ([[pictureUrl substringWithRange:NSMakeRange(0, 8)] isEqualToString:@"https://"]
+        || [[pictureUrl substringWithRange:NSMakeRange(0, 7)] isEqualToString:@"http://"]){
+        return [NSURL URLWithString:pictureUrl];
+    }
+    
+    NSString* completeString = [NSString stringWithFormat:@"%@%@%ld%@%@", KSMediaResizerBaseUrl, @"/", (unsigned long) width, @"x/", pictureUrl];
```
Comment (cgwyllie) [inline Sources/Shared/KSMediaHelper.m:21]: Maybe formatter should be `%lu` over `%ld` according to https://stackoverflow.com/a/22671499

File: Sources/KumulosInApp.m (hunk 1 above)
Comment (ftcvlad) [reply Sources/KumulosInApp.m:18]: increased to 300

File: Sources/KumulosSDKExtension/KumulosNotificationService.m (hunk 2 above)
Comment (ftcvlad) [reply Sources/KumulosSDKExtension/KumulosNotificationService.m:16]: pod includes all   s.source_files = "Sources/KumulosSDKExtension/**/*.{h,m}",  "Sources/Shared/**/*.{h,m}"

if you mean this

File: Sources/Shared/KSMediaHelper.m (hunk 3 above)
Comment (ftcvlad) [reply Sources/Shared/KSMediaHelper.m]: ld also worked, but lu is more correct. updated

**Pull request merged**